roles/arteria-siswrap-ws/defaults/main.yml

```diff
@@ -1,26 +1,33 @@
 ---
 
-# _environment can be set to e.g. "staging" on the command line. 
-# _version is then also preferably changed.
+# If deploying to staging one should call with 
+# -e deployment_environment=staging
+# One can also override the default installed version 
+# by overriding the arteria_checksum_version on the command line. 
 #
 # This will set corresponding paths and use the appropriate port. 
 arteria_siswrap_repo: https://github.com/arteria-project/arteria-siswrap.git
 arteria_siswrap_version: v1.1.0
-arteria_siswrap_environment: production
 
 arteria_install_path: "{{ root_path }}/sw/arteria"
 
 # These values will be appended with production and staging specific
 # paths in the tasks.
 #
 # NB. The log dirs need to be created manually on destination cluster.
+#
+# NB 2. Hard coded root path for runfolders, because that path 
+# needs to be reachable from outside world; i.e. have to be under /proj
+# And hard coded log paths because we do not want the log files 
+# to disappear when the wildwest directory is cleaned out with every 
+# staging sync. 
 arteria_siswrap_env_root: "{{ arteria_install_path }}/siswrap_venv"
 arteria_siswrap_sources_path: "{{ arteria_install_path }}/siswrap_src"
 arteria_siswrap_config_root: "{{ ngi_pipeline_conf }}/arteria/siswrap"
 arteria_siswrap_app_config: "{{ arteria_siswrap_config_root }}/app.config"
 arteria_siswrap_logger_config: "{{ arteria_siswrap_config_root }}/logger.config"
-arteria_siswrap_log: "{{ ngi_pipeline_upps_path }}/log/arteria/siswrap-ws"
-runfolder_path: "{{ proj_root }}/{{ ngi_pipeline_upps_delivery }}/incoming/"
+arteria_siswrap_log: "/proj/{{ ngi_pipeline_upps_delivery }}/private/log/arteria/siswrap-ws"
+runfolder_path: "/proj/{{ ngi_pipeline_upps_delivery }}/incoming/"
```

Comment:
This and a few other instances always pumps the output to /proj/ even for a staging instance. Is this intended?